configs/components/pxp-agent.rb

```diff
@@ -30,10 +55,10 @@
       "#{cmake}\
       #{toolchain} \
           -DCMAKE_VERBOSE_MAKEFILE=ON \
-          -DCMAKE_PREFIX_PATH=#{settings[:prefix]} \
-          -DCMAKE_INSTALL_PREFIX=#{settings[:prefix]} \
-          -DCMAKE_SYSTEM_PREFIX_PATH=#{settings[:prefix]} \
-          -DMODULES_INSTALL_PATH=#{File.join(settings[:install_root], 'pxp-agent', 'modules')} \
+          -DCMAKE_PREFIX_PATH=#{prefix} \
+          -DCMAKE_INSTALL_PREFIX=#{prefix} \
+          -DCMAKE_SYSTEM_PREFIX_PATH=#{prefix} \
+          -DMODULES_INSTALL_PATH=#{File.join(install_root, 'pxp-agent', 'modules')} \
```

Comment:
Oh dang, good catch. I was setting `install_root` for windows, but it looks like that was lost.